themes/bootstrap3/templates/search/list-list.phtml

```diff
@@ -1,11 +1,9 @@
 <? if (!isset($this->indexStart)) $this->indexStart = 0; ?>
-<? $showCheckboxes = (isset($this->showCartControls) && $this->showCartControls)
```

Comment:
Here's another place where showCartControls is being factored out, which may or may not be appropriate -- but it might need another look, in relation to my previous comment.